content/modules/redisai/_index.md

```diff
@@ -0,0 +1,94 @@
+| |              | |  +--------+            |             +-------------+ | | |
+| |              | +------------------------|-----------------------------+ | |
+| +--------------|--------------------------|-------------------------------+ |
+|                v                          v                                 |
+| +--------------+-----------------+   +------------------------------------+ |
+| | RAM                            |   | DEVICES                            | |
+| |                                |   | +-----+  +-----+  +-----+  +-----+ | |
+| | 00101010 00101010 00101010 ... |   | | CPU |  | GPU |  | TPU |  | ... | | |
+| |                                |   | +-----+  +-----+  +-----+  +-----+ | |
+| +--------------------------------+   +------------------------------------+ |
++-----------------------------------------------------------------------------+
+```
+
+## How RedisAI Works
+
+RedisAI bundles together best-of-breed technologies for delivering stable and performant computation graph serving.
+Every DL/ML framework ships with a runtime for executing the models developed with it, and the common practice for serving these is building a simple server around them.
+
+RedisAI aims to be that server, saving you from the need of installing the backend you're using and developing a server for it.
+By itself that does not justify RedisAI's existence so there's more to it. Because RedisAI is implemented as a Redis module it automatically benefits from the server's capabilities: be it Redis' native data types, its robust eco-system of clients, high-availability, persistence, clustering, and Enterprise support.
+
+Because Redis is an in-memory data structure server RedisAI uses it for storing all of its data.
+The main data type supported by RedisAI is the Tensor that is the standard representation of data in the DL/ML domain.
+Because tensors are stored in the memory space of the Redis server, they are readily accessible to any of RedisAI's backend libraries at minimal latency.
+
+The locality of data, which is tensor data in adjacency to DL/ML models backends, allows RedisAI to provide optimal performance when serving models.
+It also makes it a perfect choice for deploying DL/ML models in production and allowing them to be used by any application.
+
+Furthermore, RedisAI is also an optimal testbed for models as it allows the parallel execution of multiple computation graphs and, in future versions, assessing their respective performance in real-time.
+
+## Data Structures
+
+RedisAI provides the following data structures:
+
+- **Tensor**: Represents an n-dimensional array of values
+- **Model**: Represents a computation graph by one of the supported DL/ML framework backends
+- **Script**: Represents a [TorchScript](https://pytorch.org/docs/stable/jit.html) program
+
+## DL/ML Backends
+
+RedisAI supports the following DL/ML identifiers and respective backend libraries:
```

Comment:
I would remove this and link to the release notes.  This contains as section which version are currently supported.  Or state that we are continuously adding backends, so latest info is in the release notes